docs/support.rst

```diff
@@ -24,5 +24,32 @@ GPG Key Fingerprint: 6D5B EF9A DD20 7580 5747  B70F 9F88 FB52 FAF7 B393
 The public key can be fetched from any GPG keyserver, but be careful: you must
 use the **full fingerprint** to check that you got the correct key.
 
-`Releases <https://github.com/borgbackup/borg/releases>`_ are signed with this GPG key,
-please use GPG to verify their authenticity.
+Verifying signed releases
+-------------------------
+
+`Releases <https://github.com/borgbackup/borg/releases>`_ are signed with the same GPG key and a .asc file is provided for each binary.
+
+To verify a signature, the public key needs to be known to GPG. It can be imported into the local keystore from a keyserver with the fingerprint:
+
+      gpg --recv-keys "6D5B EF9A DD20 7580 5747 B70F 9F88 FB52 FAF7 B393"
+
+If GPG successfully imported the key, the output should be (among other things): 'Total number processed: 1'.
+
+To verify for example the signature of the borg-linx64 binary:
+
+      gpg --verify borg-linux64.asc borg-linux64
```

Comment:
it even works without giving the binary filename.
gpg is clever enough to just remove the .asc and use that.